### ISSUE TITLE
StaticMaps doesn't work with coordonates for Center

### ISSUE THREAD

**Issue by speedwheel** [googlemaps/google-maps-services-go#130]
I am trying to use Lat and Lng instead of address, but it doesn't work, here is my code:

```go
r := &maps.StaticMapRequest{
	Center: "44.4267674,26.102538399999958",
        Size:   "600x300",
	Zoom:   10,
}
```
This location should be in Bucharest Romania, but instead I get some other location in America.

How I solved it; in staticmap.go I changed this:

```go
if r.Center != "" {
	q.Set("center", url.QueryEscape(r.Center))
}
```
to this:
```go
if r.Center != "" {
	q.Set("center", r.Center)
}
```

**Comment by domesticmouse** [comment]
Hey @speedwheel, could you please turn this into a Pull Request? Thanks!

**Comment by speedwheel** [comment]
#131

**Comment by domesticmouse** [comment]
I've merged #131 with fixes for my tests. Thanks!
